ably/state.go

```diff
@@ -29,7 +29,7 @@ func (st StateType) String() string {
 	case StateChan:
 		return "channel"
 	default:
-		return ""
+		return "invalid"
```

Comment:
I suppose this exists because you added this fake stateGuard in your tests. Empty string looked like the valid approach though.